### PR TITLE
prepare release 2.0.4

### PR DESCRIPTION
**Pull request by chaudum**

### CODE REVIEW

File: blackbox/docs/release_notes/2.0.4.txt [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+   :ref:`full_restart_upgrade`.
+
+   Consult the :ref:`version_2.0.0_upgrade_notes` for ``2.0.0`` when
+   upgrading.
+
+.. WARNING::
+
+   If you upgrade from a version greater or equal than 2.0.0 you need to
+   perform a :ref:`rolling_upgrade` in order to prevent BLOB data loss caused
+   by a bug introduced with :ref:`version_2.0.0`.
```
Comment (joemoe) [inline blackbox/docs/release_notes/2.0.4.txt]: With replicas > 1 and the cluster always returning to a green cluster state between switching upgrading the different nodes.

File: blackbox/docs/release_notes/2.0.4.txt [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+.. WARNING::
+
+   Before upgrading, you should `back up your data`_.
+
+.. _back up your data: https://crate.io/a/backing-up-and-restoring-crate/
+
+Upgrade Notes
+=============
+
+Due to a bug introduced in :ref:`version_2.0.0` that could cause loss of BLOB
```
Comment (nslater) [inline blackbox/docs/release_notes/2.0.4.txt]: "can cause"

File: blackbox/docs/release_notes/2.0.4.txt [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+data it is necessary to perform a :ref:`rolling_upgrade` from versions greater
```
Comment (nslater) [inline blackbox/docs/release_notes/2.0.4.txt]: "data, "

File: blackbox/docs/release_notes/2.0.4.txt [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+or equal 2.0.0 if you are using BLOB tables.
+
+Additionally the number of replicas need to be set to at least 1 for **all**
```
Comment (nslater) [inline blackbox/docs/release_notes/2.0.4.txt]: "Additionally,"

Comment (nslater) [inline blackbox/docs/release_notes/2.0.4.txt]: "needs"

Comment (nslater) [inline blackbox/docs/release_notes/2.0.4.txt]: "1" should be a literal

File: blackbox/docs/release_notes/2.0.4.txt [hunk 5]
```diff
@@ -0,0 +1,74 @@
...
+blob tables and you need to make sure data is fully replicated.
```
Comment (nslater) [inline blackbox/docs/release_notes/2.0.4.txt]: "that data"

Comment (nslater) [inline blackbox/docs/release_notes/2.0.4.txt]: "replicated before upgrading"

Comment (nslater) [inline blackbox/docs/release_notes/2.0.4.txt]: "before continuing" sorry

File: blackbox/docs/release_notes/2.0.4.txt [hunk 6]
```diff
@@ -0,0 +1,75 @@
...
+   Before upgrading, you should `back up your data`_.
+
+.. _back up your data: https://crate.io/a/backing-up-and-restoring-crate/
+
+Upgrade Notes
+=============
+
+Due to a bug introduced in :ref:`version_2.0.0` that can cause loss of BLOB
+data, it is necessary to perform a :ref:`rolling_upgrade` if you're running a
+version greater or equal 2.0.0 and using BLOB tables.
```
Comment (nslater) [inline blackbox/docs/release_notes/2.0.4.txt]: "greater than or equal to"

**Pull request merged**